app/policies/birth_record_policy.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class BirthRecordPolicy < ApplicationPolicy
+  def show?
+    true
```

Comment:
Any better suggestions for rules here? @Br3nda

Reply:
```user.documents.includes @record```